extensions/google-chrome/src/google-chrome-new-tab.tsx

```diff
@@ -0,0 +1,229 @@
+  
+  const script = `
+    tell application "Google Chrome"
+      activate
+      tell window 1
+          set newTab to make new tab ` + (queryText ? 'with properties {URL:"http://www.google.com/search?q='+queryText+'"}' : '') + ` 
+      end tell
+    end tell
+  `
+
+  return await runAppleScript(script);
+}
+
+
+
+
+async function setActiveTab(tab: Tab): Promise<void> {
+  await runAppleScript(`
+    tell application "Google Chrome"
+      activate
+      set index of window (${tab.windowsIndex} as number) to (${tab.windowsIndex} as number)
+      set active tab index of window (${tab.windowsIndex} as number) to (${tab.tabIndex} as number)
+    end tell
+  `);
+}
+
+interface State {
+  tabs?: Tab[];
+}
+
+
+export default function Command(): ReactElement {
+    const [searchText, setSearchText] = useState<string>()
+    const { isLoading, error, entries } = useChromeHistorySearch(searchText)
+
+
+    const { useOriginalFavicon } = getPreferenceValues<{ useOriginalFavicon: boolean }>();
+    const [state, setState] = useState<State>({});
+
+    async function getTabs(query: string | null) {
+      let tabs = await getOpenTabs(useOriginalFavicon);
+
+      if(query){
+        tabs = tabs.filter(function (tab) {
+          return (tab.title.toLowerCase().includes(query.toLowerCase()) || tab.urlWithoutScheme().toLowerCase().includes(query.toLowerCase()))
+        })
+      }
+      setState({ tabs: tabs });
+    }
+
+    useEffect(() => { 
+      getTabs(null);
+    }, []);
+
+    if (error) {
+        showToast(ToastStyle.Failure, "An Error Occurred", error.toString())
+    }
+
+    return (
+        <List onSearchTextChange={function (query) { setSearchText(query); getTabs(query);}} isLoading={isLoading} throttle={false}>
+            <List.Section title="New Tab" key="new-tab">
+              <List.Item
+                title={(!searchText ? "Open empty tab" : "Search \""+searchText+"\"")}
+                icon={{ source: (!searchText ? Icon.Plus : Icon.MagnifyingGlass)}}
+                actions={<NewTabActions query={searchText} />} 
+              />
+            </List.Section>
+            <List.Section title="Open Tabs" key="open-tabs">
+              {state.tabs?.map((tab) => (
+                <TabListItem key={tab.key()} tab={tab} useOriginalFavicon={useOriginalFavicon} />
+              ))}
+            </List.Section>
+            <List.Section title="Recently Closed" key="recently-closed">
+                {entries?.map((e) => (
+                    <HistoryItem entry={e} key={e.id} />
+                ))}
+            </List.Section>
+        </List>
+    )
+}
+
+
+
+const NewTabActions = (props: { query: string | undefined }): ReactElement => {
+    const query = props.query
+
+    return (
+        <ActionPanel title="New Tab">
+          <ActionPanel.Item
+            onAction={function () { openNewTab(query)}}
+            title={(query ? "Search \""+query+"\"" : "Open empty tab")} />
```

Comment:
```suggestion
            title={(query ? `Search "${query}"` : "Open Empty Tab")} />
```

Corrected naming and added better readability